docs/api.md

```diff
@@ -117,12 +119,17 @@ It also ensures that the scroll event is propagated properly for parent ScrollVi
 
 The `SpatialNavigationScrollView` component receives the following props:
 
-| Name              | Type        | Default | Description                                                                                                                                   |
-| ----------------- | ----------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
-| `horizontal`      | `boolean`   | `false` | Determines if the scrolling orientation is horizontal. If `false`, the scrolling orientation will be vertical.                                |
-| `offsetFromStart` | `number`    | `0`     | This offset is used to prevent the element from sticking too closely to the edges of the screen during scrolling. This is a margin in pixels. |
-| `style`           | `ViewStyle` | `null`  | Style for the ScrollView. This can be any valid React Native style object.                                                                    |
-| `children`        | `ReactNode` | `null`  | Child elements of the component. They are expected to be one or multiple `SpatialNavigationNode` elements.                                    |
+| Name                            | Type           | Default | Description                                                                                                                                               |
+| ------------------------------- | -------------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `horizontal`                    | `boolean`      | `false` | Determines if the scrolling orientation is horizontal. If `false`, the scrolling orientation will be vertical.                                            |
+| `offsetFromStart`               | `number`       | `0`     | This offset is used to prevent the element from sticking too closely to the edges of the screen during scrolling. This is a margin in pixels.             |
+| `style`                         | `ViewStyle`    | `null`  | Style for the ScrollView. This can be any valid React Native style object.                                                                                |
+| `children`                      | `ReactNode`    | `null`  | Child elements of the component. They are expected to be one or multiple `SpatialNavigationNode` elements.                                                |
+| `ascendingArrow`                | `ReactElement` | `null`  | Optional component to display as the arrow to scroll on the ascending order. Used only with pointers.                                                     |
```

Comment:
You could prefix your descriptions with this?

```suggestion
| `ascendingArrow`                | `ReactElement` | `null`  | For web TVs cursor handling. Optional component to display as the arrow to scroll on the ascending order. Used only with pointers.                                                     |
```

Reply:
Done too